src/providers/comment/comment.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore, AngularFirestoreCollection } from 'angularfire2/firestore';
+import { Comment } from '../../models/comment';
+import { Observable } from 'rxjs/Observable';
+import * as firebase from 'firebase';
+
+@Injectable()
+export class CommentProvider {
+  private commentsCollection: AngularFirestoreCollection<Comment>;
+  private comments: Observable<Comment[]>;
```

Comment:
The value of `comments` is never read. Should it be removed?

Reply:
I think it should be removed. The patient-detail.ts is where the observable is used for comments on a patient view, not necessary to have it here as well (?)